public_html/wp-content/plugins/camptix-attendance/addons/assets/attendance-ui.js

```diff
@@ -435,6 +441,13 @@ jQuery(document).ready(function($){
 			this.on( 'filter', this.filter, this );
 
 			this.setupCollection();
+
+			if (
+				_camptixAttendanceQRScanning !== undefined &&
+				_camptixAttendanceQRScanning
+			) {
+			}
```

Comment:
Nothing is happening in the body of this condition. Is the `this.qr();` call meant to be in the body?

Reply:
exactly, probably for testing in the wrong spot. This initialize the qr code if the settings is enabled